projects/ad40xx_fmc/zed/system_bd.tcl

```diff
@@ -21,5 +22,7 @@ ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 9
 ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "[pwd]/mem_init_sys.txt"
 ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 9
 
-sysid_gen_sys_init_file
+set AD40XX_ADAQ400X_N $::env(AD40XX_ADAQ400X_N)
```

Comment:
What happens if the AD40XX_ADAQ400X_N is not set in the environment ? 
Maybe we should use the get_env_param  with default value as in the system_project.tcl